Arithmetic.pp

```diff
@@ -65,7 +66,10 @@
     ternary() ( ::times:: #multiplication expression() )?
 
 ternary:
-    term() ( ::div:: #division expression() )?
+    quaternary() ( ::div:: #division expression() )?
+
+quaternary:
+    term() ( ::pow:: #power expression() )?
```

Comment:
Ok, it seem my mind and my college calculator were wrong ;)
https://en.wikipedia.org/wiki/Order_of_operations#Special_cases